aws-cli-mfa

```diff
@@ -129,19 +129,22 @@ RESPONSE=$(COLUMNS=999 /usr/bin/env python3 -c "$PYCODE" $@)
 
 if [[ $RESPONSE == usage* ]] ; then
     echo "$RESPONSE"
-    exit
+    return 2>/dev/null || exit
```

Comment:
https://stackoverflow.com/questions/9640660/any-way-to-exit-bash-script-but-not-quitting-the-terminal/54344104#54344104